test/library/standard/Path/victor-ludorum/cygwin/testcases_isAbsPath.chpl

```diff
@@ -0,0 +1,19 @@
+   use Path;
```

Comment:
I'm not seeing the skipif so that this test only runs on cygwin.  There should be two skipif files: 1) the one for the Linux and Mac environment (which you have), and one for this test that lives in the same directory